datafusion/core/src/datasource/physical_plan/parquet/row_groups.rs

```diff
@@ -657,7 +664,7 @@ mod tests {
         use datafusion_expr::{col, lit};
         // test row group predicate with an unknown (Null) expr
         //
-        // int > 1 and bool = NULL => c1_max > 1 and null
+        // c1 > 15 and c2 = NULL => c1_max > 15 and null
```

Comment:
nit (but as take this chance to change it):

```suggestion
        // c1 > 15 and c2 = NULL => c1_max > 15 and NULL
```